crates/gateway-types/src/reply.rs

```diff
@@ -679,10 +679,12 @@ pub mod state_update {
         pub old_declared_classes: Vec<ClassHash>,
```

Comment:
This should be called `old_declared_contracts`. Tested with https://external.integration.starknet.io/feeder_gateway/get_state_update?blockHash=0x067D24961077DE7A1F49B6B1DA78903EE80217662D026C155E14D7ABEDC04034

I think we should add proper fixtures and tests for this change. I suggest state updates for integration blocks 283363 (for `declared_classes`) and 283428 (for `replaced_classes`).

Reply:
Sorry, I've just realized that it's block 283364 that contains a class in `declared_classes`...